integration/tests/command_line/deploy_modules_with_envs.rb

```diff
@@ -0,0 +1,51 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'CODEMGMT-90 - C62418 - Deploy Modules with Environments'
```

Comment:
This test doesn't make any sense as currently designed. We should re-design the test case.